languages/gutenberg.pot

```diff
@@ -7,6 +7,7 @@ msgstr ""
 msgid "Embed"
 msgstr ""
 
+#: blocks/library/embed/index.js:33
```

Comment:
Appears the i18n tools are misbehaving again; I think the reference on the following line was meant to be removed. `npm run build` should clear it up.